gui/wxpython/wxgui.py

```diff
@@ -100,10 +100,10 @@ def show_main_gui():
                     warning(
                         _(
                             "Current version of wxPython {} is lower than "
-                            "minimum required version {}".format(
-                                wx.__version__,
-                                ".".join(map(str, min_required_wx_version)),
-                            )
+                            "minimum required version {}"
+                        ).format(
+                            wx.__version__,
+                            ".".join(map(str, min_required_wx_version)),
                         )
                     )
                 else:
```

Comment:
Here too, multiple placeholders that aren't numbered or named, so maybe it is impossible to translate correctly the string because the placeholders can't be reordered for the a specific language.

Reply:
This is the case also for several places with % formatting and `%s`. These will need to be changed at some point.

Reply:
For later